### DISCUSSION TITLE
Ownership/naming of this repo

### DISCUSSION THREAD

**Discussion by isaackehle** [ncstate-sat/popover#255] [Announcements]
Hello everyone.  

I apologize for the long delay and lack of support overall for this repo.  There are many users of this tool, and from an Open Source community perspective, it's sometimes a struggle to devote any time to it.  I am no longer with NC State, nor is Will who was the major contributor here.  The department we worked for, NC State SAT, has no interest in maintaining this repo (and likely don't even know it exists). 

When we first started the project, we named it under `@sat` (https://www.npmjs.com/package/@sat/popover). Then we renamed the organization to `@ncstate` (https://www.npmjs.com/package/@ncstate/sat-popover) to be under that umbrella.  

Since we are both no longer here, and there have been a few contributors, I'd like to open this up more to the community.  That would require deprecating it from `@ncstate` to something else.  Perhaps something I make up, or just my `@isaackehle`. 

Please give some thoughts and suggestions, and if you'd like to be a maintainer. Thanks!

**Comment by MatthewMarichiba** [comment]
Hi! Let me start with a big THANK YOU for the work you've done on the sat-popover component. Great work here! It has been really beneficial to my team, and I appreciate you for open-sourcing your efforts.

I don't have a preference about where the ownership lies. Speaking for my team, we would be fine with it housed under your personal account. We use several packages by individual package owners, without particular concern that there is no broader organizational sponsorship, as long as the package is of good repute and is regularly maintained. Just to speak my greedy bias: Whatever is the fastest path to merging PR #262 for Angular 14 support (and future ng releases), then I think _that's_ the right path! :)

I don't have experience as a maintainer for npm packages. I don't particularly want to take on a side-project or commit to feature development, but if it allows me to periodically bump up Angular versions in the package.json and run the build & release scripts (or in the present case, just approve the work already done in #262 ), I could be up for that.

Thanks & regards, -mm

**Comment by isaackehle** [comment]
Thanks so much for responding @MatthewMarichiba .

I think that if there were more people able to give approvals on PRs, this would have more traction and development right now.  I didn't have to be the one to give a thumbs up to do the merge, but I did have to be the one to do a new release.  That all said, even with the PR being more complicated than just a package update, others having a say in the continued improvement of this tool is a good thing.

So that said, I'd be happy to give you access to be an approver if you are willing.

I also thought that this project's major revision number could be bumped up to align with the major revision number of Angular.

Additionally, can consider transferring it to my personal account.  I'm not quite sure when the right time would be to make that happen, maybe along side the release of v14.  I think it would make sense to change the plugin name to be a more generic name, and please send out some options!

Thanks everyone.  v10.0.0 is available as of last night!

**Comment by MatthewMarichiba** [comment]
Thanks so much for pushing the v10 release! My own team's project is now up-to-date with Ng14, thanks to you @isaackehle .

I'm happy for you to add me as an approver. I can't promise to commit much time, but I'd be happy to have experience playing a role on an open-source project, and we'll see where it goes.

In terms of direction & broader adoption, here are my random thoughts in response to your comments:
* It seems like aligning with the Angular releases is a common approach for package publishers, and as a developer and consumer of packages, I find it an intuitive approach.
* In terms of a better name to improve adoption, I will say that *my team* found this component while searching for a tooltip that can render HTML templates inside the tooltip. `sat-popover` has other cool uses that I'm actually excited to incorporate, but at the moment we're using it just as a hover tooltip with HTML tables inside. My understanding is that in the UX world "popover" and "tooltip" mean different things, and `sat-popover` definitely does more than just a tooltip, but I think more people immediately recognize the term "tooltip". Anyway, just a thought for framing the tool to increase search relevance. "`sat-popover`" lacks a certain ring of "That's exactly what I need!"  :)
* It looks like "the obvious" names are taken: `ng-popover`, `ngx-popover`, `ngx-smart-popover`. If this package had been called `ngx-html-tooltip`, I probably would have found it immediately. If the popover-ness is important to you, maybe `ngx-html-popver` or `ngx-template-popover`? (I notice that some of the other "popover" packages only accept content for static fields, which is a differentiator for you).

**Comment by isaackehle** [comment]
Welcome @staylor8-ncsu to the discussion.  Since this started with NC State's SAT Department, and Shawn came back to the university, I am happy to defer this to his vision.

**Reply by staylor8-ncsu** [reply]
Thanks @isaackehle for everything you have done and are doing with this component. I'm not able to help with the discussion about versioning, js code or anything really as it relates to the FE. UI is something I am just not good at.

I'm happy to have it continue to live here if that makes it easier/simpler or provides some value to the community using it to not have to fund anything to maintain and use it or more broadly share it.

Tell me what you want or need, and we'll make it happen collectively.

**Reply by isaackehle** [reply]
@staylor8-ncsu, it's great you reached out and have been making this org more active.  I think given that, keeping it where it is for now, with other eyes helping to keep things fresh, makes life easier for everyone.

**Reply by wjaspers** [reply]
Should this project be updated to indicate it is only in maintenance, or is looking to be transferred?

**Comment by Andrew-Marks-Trisept** [comment]
This package is great -- as I [mentioned in the popover topic on the Angular repo](https://github.com/angular/components/issues/2691#issuecomment-1147670900), it's the only one I've found that's actually keyboard accessible.

The only thing I really need at this point is for it to be updated with each new Angular release so I don't get "incompatible peer dependency" errors.  There's an open issue here: https://github.com/ncstate-sat/popover/issues/270

@isaackehle or @staylor8-ncsu is this something you can help get approved?

**Reply by wjaspers** [reply]
Unfortunately NPM's handling of peer dependencies has made it more difficult to honor open ended minor or patch version changes. I'd be happy to keep my fork alive for the strict purpose of Angular version updates.

**Reply by isaackehle** [reply]
The repo has been updated. Thanks for your patience.
